src/main/java/com/deloitte/ddwatch/controllers/ProjectReportController.java

```diff
@@ -0,0 +1,57 @@
+package com.deloitte.ddwatch.controllers;
+
+import com.deloitte.ddwatch.dtos.MetricsReportDTO;
+import com.deloitte.ddwatch.model.MetricsReport;
+import com.deloitte.ddwatch.model.Project;
+import com.deloitte.ddwatch.services.ProjectReportService;
+import com.deloitte.ddwatch.services.ProjectService;
+import lombok.extern.slf4j.Slf4j;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
+@CrossOrigin
+@RestController
+@RequestMapping("/project_report")
```

Comment:
we already have a mapping for "/projects" and this is the delivery report component so you can use something like @RequestMapping("/projects/{id}/deliveryReports") (a delivery report is part of a project)